src/test/java/io/nats/client/impl/JetStreamGeneralTests.java

```diff
@@ -972,7 +972,12 @@ public void testGetJetStreamValidatedConnectionCoverage() {
         IllegalStateException ise = assertThrows(IllegalStateException.class, njsm::getJetStreamValidatedConnection);
         assertTrue(ise.getMessage().contains("subscription"));
 
-        njsm.subscription = new NatsSubscription("sid", "sub", "q", null, null);
+        // make a dummy connection so we can make a subscription
+        Options options = Options.builder().build();
+        NatsConnection nc = new NatsConnection(options);
+        njsm.subscription = new NatsSubscription("sid", "sub", "q", nc, null);
+        // remove the connection so we can test the coverage
+        njsm.subscription.connection = null;
```

Comment:
Needed b/c of the MessageQueue constructor change, because code now requires the options. This was a coverage test anyway, this should NEVER happen in real execution.